pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -380,6 +380,18 @@ func uploadAndStart(nodes option.NodeListOption, v string) versionStep {
 	}
 }
 
+// uploadAndStartSequential is like uploadAndStart, but starts node sequentially
+// to bootstrap the cluster with deterministic node IDs.
```

Comment:
Why don't you start from fixtures, like the other mixed-version tests? I doubt that really adds anything new for this particular test (since we're mostly operating on range descriptors split using the current version) but as the fixtures get better this test will.

Reply:
I though the checkpoint stuff were all about improving setup times, which isn't relevant here, but I understand now that they're used to start the clusters out with some historical state.